server/service/src/vaccination/get_vaccination_card.rs

```diff
@@ -123,6 +137,36 @@ pub fn get_suggested_date(
     }
 }
 
+pub fn get_vaccination_status(
+    row: &VaccinationCardRow,
+    patient_dob: Option<NaiveDate>,
+) -> Option<VaccinationCardItemStatus> {
+    if row.given == Some(true) {
+        Some(VaccinationCardItemStatus::Given);
+    }
+    if row.given == Some(false) {
+        Some(VaccinationCardItemStatus::NotGiven);
+    }
+
+    if patient_dob == None {
+        return None;
+    }
+
+    let patient_age_in_months =
+        ((Local::now().date_naive() - patient_dob.unwrap()).num_days() as f64) / MONTHS_PER_YEAR;
```

Comment:
Two things here! 

All our backend date calculations should use Utc, so dates are consistent when we sync.

For the `paitient_dob` - while it should be safe given you have done the `None` check above, we pretty strictly don't use `.unwrap` in production code, as it could panic if it fails. If you need to unwrap something, it's recommended to use `.unwrap_or(..)` to provide a fallback. There's a number of other ways around it e.g. `match` statement, `.map()`, `if let Some()`.. example below!

Reply:
Oo great to know, thank you! Yeah when working with months, I think we're pretty safe 😁